bears/perl/PerlCriticBear.py

```diff
@@ -1,36 +1,33 @@
-import re
-
-from coalib.bearlib.abstractions.Lint import Lint, escape_path_argument
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+import shutil
```

Comment:
@Adrianzatreanu When I started with coala, I was said that I should specify at least a line gap between coala modules and non-coala modules. :smile: